pci-dss/system/ksp-pci-dss-8.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   selector:
     matchLabels:
       pod: test  #change app: test to match your label
-  file:
+  process:
     severity: 5
     matchPaths:
     - path: /var/log/utmp
```

Comment:
For this path, I'm wondering 🤔 how this could be process, since Linux File System normally stored file in this path

Reply:
When we have `file` object declared two times, we have some issue with parsing. But editing `file` to `files` makes the job. BTW I'm not sure about the template. Thanks